x-pack/plugins/reporting/common/constants.js

```diff
@@ -18,4 +18,4 @@ export const UI_SETTINGS_CUSTOM_PDF_LOGO = 'xpackReporting:customPdfLogo';
  * The type name used within the Monitoring index to publish reporting stats.
  * @type {string}
  */
-export const KIBANA_REPORTING_TYPE = 'reporting_stats';
+export const KIBANA_REPORTING_TYPE = 'reporting';
```

Comment:
This change makes `reporting` the field name of the reporting usage data in the `/api/stats` response. The change doesn't affect internal upload because the name is hardcoded in `BulkUploader.combineStatsLegacy`

Reply:
Based on this comment I'm confused why the constant is used in https://github.com/tsullivan/kibana/blob/ab67e641c93f20d2df59eb6bbb47de887ae4f5ac/x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.combine_stats_legacy.test.js#L50